### DISCUSSION TITLE
Updating the native version

### DISCUSSION THREAD

**Discussion by stephen1977** [riderx/capacitor-updater#133] [Plugin]
can this plugin update the native version. i..e the version and build number. ive been trying to get it to work with no luck

   const version = await CapacitorUpdater.download({ url: updateUrl, version: newVersion});
    alert(JSON.stringify(version));
    await CapacitorUpdater.set( {id: version.id}); // sets the new version, and reloads the app

its does update the codebase but the version number remains the same. as you can see a new version is required as a param for the the download function and is also returned in the bundle. any help would be much appreciated. thanks

**Comment by riderx** [comment]
Hello how do you get the native version ? if it's with capacitor api, you will not get different version number since that the native and you can't update it, instead do a get Current form the plugin https://github.com/Cap-go/capacitor-updater#current
